CLAUDE.md

```
@@ -0,0 +1,83 @@
+npm run test:config    # build + verify config schema
+npm run start:container  # docker compose up
+```
+
+Debug logging:
+```bash
+DEBUG=moneyman:* npm run start
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)